.env.local.example

```diff
@@ -1,8 +1,13 @@
 # Clerk. See https://clerk.com
 NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY=
 CLERK_SECRET_KEY=
+# DO NOT modify
+NEXT_PUBLIC_CLERK_SIGN_IN_URL=/signin
+NEXT_PUBLIC_CLERK_SIGN_UP_URL=/join
+NEXT_PUBLIC_CLERK_AFTER_SIGN_IN_URL=/
+NEXT_PUBLIC_CLERK_AFTER_SIGN_UP_URL=/
```

Comment:
Do we want this to be the home page or should it be `/settings` in the membership tab so that people have direct access to pay?

Reply:
aftersignin here is not the main logic. if you want this, we have to do it code

Reply:
this is for, like google login, redirect url

Reply:
right, okay